src/NzbDrone.Core/Movies/MovieRepository.cs

```diff
@@ -182,23 +181,27 @@ private Movie FindByTitle(string cleanTitle, int? year)
             
             if (result == null)
             {
-                result =
-                    Query.Where(movie => movie.CleanTitle == cleanTitleWithArabicNumbers).FirstWithYear(year) ??
-                    Query.Where(movie => movie.CleanTitle == cleanTitleWithRomanNumbers).FirstWithYear(year);
+                result = Query.Where(movie => movie.CleanTitle == cleanTitleWithArabicNumbers || movie.CleanTitle == cleanTitleWithRomanNumbers)
+                    .FirstWithYear(year);
 
                 if (result == null)
                 {
-
-                    result = Query.Join<Movie, AlternativeTitle>(JoinType.Inner, m => m.AlternativeTitles, (m, t) => m.Id == t.MovieId)
-                                  .Where(t => t.CleanTitle == cleanTitle || t.CleanTitle == cleanTitleWithArabicNumbers || t.CleanTitle == cleanTitleWithRomanNumbers)
+                    result = Query.Where<AlternativeTitle>(t => t.CleanTitle == cleanTitle || t.CleanTitle == cleanTitleWithArabicNumbers || t.CleanTitle == cleanTitleWithRomanNumbers)
                                   .FirstWithYear(year);
-
                 }
             }
 
             return result;
         }
 
+        public List<Movie> FindByTitleInexact(string cleanTitle)
+        {
+            var mapper = _database.GetDataMapper();
```

Comment:
This does not properly close the data mapper. But then again, this is still based on the old develop branch, where every function here does that :)

It would be best to do:
```c#
using (var mapper = _database.GetDataMapper())
{
    ...
}
```

Or even better, once develop is merged back into aphrodite, use `Query()` directly.

Reply:
I may have accidentally reverted that in places when pulling some Sonarr stuff.